digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/pointasset/OraclePedestrianCrossingDao.scala

```diff
@@ -77,6 +78,15 @@ class OraclePedestrianCrossingDao() {
     id
   }
 
+  def fetchByRadius(position : Point, meters: Int): Seq[PedestrianCrossing] = {
```

Comment:
This fetchByRadius method propagated in these new points can be restructured some part to pointAssetOperations, see the all-new fetchByRadius methods to apply this suggestion